CSharp/BatchExplorer/Helpers/Common.cs

```diff
@@ -40,14 +40,26 @@ public static class Common
                 "A7",
                 "A8",
                 "A9",
+                "A10",
+                "A11",
                 "STANDARD_D1",
+                "STANDARD_D1_V2",
                 "STANDARD_D2",
+                "STANDARD_D2_V2",
                 "STANDARD_D3",
-                "STANDARD_D4",
```

Comment:
Did you intend to remove `STANDARD_D4`? I thought Batch supported all the Dv1 and Dv2 VM sizes.